packages/atomic/src/components/search/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -36,27 +41,50 @@ export class AtomicResultImage implements InitializableComponent {
     return Array.isArray(value) ? value[0] : value;
   }
 
+  private handleImageError() {
+    let message =
+      'The image url is not valid. You might want to add a "fallback" property.';
+
+    if (this.fallback) {
+      message = 'The image url is not valid or could not be loaded.';
+      this.host.querySelector('img')!.src = this.fallback;
+    }
+    this.bindings.engine.logger.error(message, this.host);
+  }
+
   public render() {
-    const url = this.url;
+    let url = this.url;
 
     if (!url) {
-      this.host.remove();
-      return;
+      if (this.fallback) {
+        this.bindings.engine.logger.error(
```

Comment:
Could this be a warning if the fallback is present?

Reply:
yes good idea, I did not know about the warning function 😁